spring-integration-ip/src/main/java/org/springframework/integration/ip/udp/UnicastReceivingChannelAdapter.java

```diff
@@ -73,6 +73,11 @@ public UnicastReceivingChannelAdapter(int port, boolean lengthCheck) {
 	}
 
 	@Override
+	public boolean isLongLived() {
+		return true;
+	}
```

Comment:
Why is it here, rather than in that `AbstractInternetProtocolReceivingChannelAdapter` superclass?